crates/cairo-annotations/Cargo.toml

```
@@ -27,3 +27,8 @@ regex.workspace = true
 [dev-dependencies]
 assert_fs.workspace = true
 snapbox.workspace = true
+
+[features]
+default = ["cairo-lang"]
+cairo-lang = ["cairo-lang-sierra-to-casm", "cairo-lang-sierra"]
```

```suggestion
cairo-lang = ["dep:cairo-lang-sierra-to-casm", "dep:cairo-lang-sierra"]
```
afaik that's the correct syntax https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies